packages/rule-engine/src/models/rule.ts

```diff
@@ -178,7 +189,7 @@ export interface SendEmailAnyoneValue {
   to: string[];
 }
 
-export interface UserConditionValue {
+export interface PropertiesConditionValue {
```

Comment:
Yes since we are using the same model for both User and Conversation properties we have decided to use a common name for them both
cc: @Prabhu-Kathiresan